bigquery_etl/backfill/parse.py

```diff
@@ -93,36 +93,54 @@ def __str__(self):
 
     @entry_date.validator
     def validate_entry_date(self, attribute, value):
-        """Check that provided entry date is valid."""
+        """Check that provided entry date is not in the future."""
```

Comment:
Maybe we should use this elsewhere at some point 😅 we _could_ add this as a clause to "schedule" the backfill

Reply:
We mostly use the `entry_date` as an identifier right now, to mark the tables and so on but since it's a date we could actually use it for date-like things like scheduling